qutebrowser/utils/urlutils.py

```diff
@@ -102,6 +102,13 @@ def _get_search_url(txt):
         engine = 'DEFAULT'
     template = config.val.url.searchengines[engine]
     url = qurl_from_user_input(template.format(urllib.parse.quote(term)))
+
+    if config.val.url.open_base_url and \
+            term in config.val.url.searchengines.keys():
```

Comment:
No need for the `.keys()`, iterating over a dictionary gives you its keys (and thus you can also do `key in some_dict`). With that, it also fits on one line :wink: